langlib/langlib-test/src/test/resources/test-src/floatlib_test.bal

```diff
@@ -415,6 +415,176 @@ function testFromStringNegative() {
     }
 }
 
+function testRound() {
+    assertEquality(float:round(123.123f, 2), 123.12f);
```

Comment:
Let's add a few tests with the method call syntax also.

Reply:
Also with named arguments.

Reply:
Added